js/helper.js

```diff
@@ -222,7 +250,7 @@ function prepareCommand(command, schema, options) {
  * @returns {string}
  */
 function getPath() {
-  return binaryPath;
+  return mockedBinaryPath !== undefined ? mockedBinaryPath : getBinaryPath();
```

Comment:
```suggestion
  return mockedBinaryPath || getBinaryPath();
```

Reply:
I am intentional with the undefined check here because of empty strings.